test/xcm/statemine.transfer.test.ts

```diff
@@ -0,0 +1,44 @@
+import { BN, BN_FIVE, BN_TEN } from "@polkadot/util";
+import _ from "lodash";
+import { AssetId, ChainId } from "../../utils/ChainSpecs";
+import { waitForEvent, waitForEvents } from "../../utils/eventListeners";
+import { StatemineNode } from "../../utils/Framework/Node/StatemineNode";
+import { alice, api, setupApi, setupUsers } from "../../utils/setup";
+import { signSendSuccess } from "../../utils/sign";
+import { XToken } from "../../utils/xToken";
+
+describe("XCM transfers", () => {
+    let statemine: StatemineNode
+
+    beforeAll(async () => {
+        // devops/chopsticks/statemine.yml port
+        statemine = await StatemineNode.create("ws://127.0.0.1:9948", ChainId.Statemine)
+        await setupApi()
+        setupUsers()
+    })
+
+    // todo repeat for every other asset
+    it("send USDt to mangata and back", async () => {
+        // const op = statemine.xTokenTransfer(ChainId.Mg, AssetId.USDt, AssetId.USDt.unit.mul(BN_TEN), alice)
```

Comment:
why this commented code?

Reply:
testing xcm for not yet opened channels on kusama